README.md

```diff
@@ -634,6 +634,12 @@ Sets `docker-compose` to run with `--verbose`
 
 The default is `false`.
 
+### `quiet-pull` (optional, run only)
+
+Sets `docker-compose up` to run with `--quiet-pull`. There are other commands that can accept `--quiet-pull` or equivalents. They have not been implemented yet. 
+
+The default is `false`.
```

Comment:
As the `up` command is only used when starting dependencies, I would re-word this based on that and remove the additional clarification:

```suggestion

Start up dependencies with `--quiet-pull` to prevent even more logs during that portion of the execution.

The default is `false`.
```